### DISCUSSION TITLE
Test failure while building

### DISCUSSION THREAD

**Discussion by HenryHeffan** [osmandapp/OsmAnd#16754] [General]
I am having trouble building OsmAnd successfully. The native code compiles, and most of the java code compiles, but I get a few test failures. For example, 
```
testSearch[12: poi_biergarten]

org.junit.ComparisonFailure: expected:<Biergarten ([Food])> but was:<Biergarten ([Sustenance])>
```

I have not altered the code. Any idea what I have done wrong?
Many thanks!

**Comment by vshcherb** [comment]
That strange cause our CI still works OK but may be you can disable that test locally

**Reply by sonora** [reply]
Weird, looks like I experience that same issue with 4 tests since yesterday, 20:45. An hour before it still worked, no significant code or script change I am aware of at the time:
```
> Task :OsmAnd-java:test

net.osmand.search.SearchUICoreTest > testSearch[6: fuel_diesel] FAILED
    org.junit.ComparisonFailure at SearchUICoreTest.java:218
net.osmand.search.SearchUICoreTest > testSearch[12: poi_biergarten] FAILED
    org.junit.ComparisonFailure at SearchUICoreTest.java:218
net.osmand.search.SearchUICoreTest > testSearch[25: nearest_pois] FAILED
    org.junit.ComparisonFailure at SearchUICoreTest.java:218
net.osmand.search.SearchUICoreTest > testSearch[30: poi_fuel] FAILED
    org.junit.ComparisonFailure at SearchUICoreTest.java:218

308 tests completed, 4 failed
```
PS: https://github.com/osmandapp/OsmAnd/issues/10160#issuecomment-831816505Zhis seems to not solve it.

**Comment by sonora** [comment]
Seems fixed after https://github.com/osmandapp/OsmAnd/commit/62369444e5443c685aa01f56d773da1c60c20fa5 . Thanks!
